extensions/amp-selector/0.1/amp-selector.js

```diff
@@ -106,10 +106,7 @@ export class AmpSelector extends AMP.BaseElement {
     this.init_();
     if (!this.isDisabled_) {
       this.element.addEventListener('click', this.clickHandler_.bind(this));
-      if (this.kbSelectMode_ != KEYBOARD_SELECT_MODES.NONE) {
```

Comment:
@aghassemi - yes i do understand that, AMP has a way to detect if a device has a keyboard attached or not
@kmh287  - search for `amp-mode-keyboard-active` and you should be able to track it down.